axelrod/tournament.py

```diff
@@ -27,6 +27,10 @@ def __init__(self, players, tournament_type=round_robin, name='axelrod',
             an axelrod Match object
         name : string
             A name for the tournament
+        clone_opponents : boolean
+            Whether the players should be cloned to form the opposing players.
+            Set this to True if the tournament includes matches between players
```

Comment:
It's `True` by default, so perhaps this should be "Set this to False if...", also, if this is staying in can we move it to the end of the named argument list? Only that if it's so early in the list it might feel like it's something you're 'supposed' to play with but really it's only going to be used in 'specific cases'.